bitnami/keycloak/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/bitnami-docker-keycloak
   - https://github.com/keycloak/keycloak
-version: 1.4.3
+version: 2.1.0
```

Comment:
the actual version is 2.0.1, could you check for 2.0.2?

Reply:
Hello @Mauraza,

You are right, actual is 2.0.1.
My MR relates to an improvement not a bugfix. That's the reason I propose 2.1.0.

Tell me.

Reply:
right, we leave it as a minor